src/quo2/components/profile/showcase_nav/component_spec.cljs

```diff
@@ -0,0 +1,37 @@
+(ns quo2.components.profile.showcase-nav.component-spec
+  (:require [quo2.components.profile.showcase-nav.view :as view]
+            [reagent.core :as reagent]
+            [test-helpers.component :as h]))
+
+(def generate-previews
+  (->> (range 5)
+       (map inc)
+       (mapv (fn [index]
+               {:icon (rand-nth [:i/recent :i/profile :i/communities :i/wallet :i/nft :i/token
```

Comment:
do we want random data in our tests? 🤔

Reply:
The component can display any Icon. So, I used random data.

Reply:
yeah sure, I understand it won't affect the test. However unless we explicitly need random data for these tests I think it is generally a better approach to have consistent data in our tests.

Reply:
Right. I will update the tests.

Reply:
Updated the tests 👍